trillian/migrillian/main.go

```diff
@@ -143,6 +134,52 @@ func main() {
 	}
 }
 
+// getConfig returns the migration config loaded from a file, if specified, or
+// constructs one from flags. The config is validated.
+func getConfig() (*configpb.MigrationConfig, error) {
+	if len(*cfgPath) != 0 {
+		return core.LoadConfigFromFile(*cfgPath)
+	}
+
+	cfg := &configpb.MigrationConfig{
```

Comment:
Is the extra complexity (potential for confusing invalid flag combinations etc.) worth the ability to configure either by file or by flags?

Reply:
I would prefer dropping the flags. Those are useful only for testing, but I think writing a short config file can be more enjoyable. Also, multi-tenant Migrillian will have to be configured from a file to avoid flags mess.

Reply:
Removed the flags.

Reply:
:+1: